components/authz-service/server/v2/migration_internal_test.go

```diff
@@ -2239,6 +2239,21 @@ func TestV1PolicyMigration(t *testing.T) {
 				}),
 			),
 		},
+		"service_groups,list": {
```

Comment:
ℹ️ Converting a custom v1 policy for service groups into a custom v2 policy.

Reply:
🏆